<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNS_Schedule_Info_Widget</class>
 <widget class="QWidget" name="HNS_Schedule_Info_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="GRB_Schedule">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>691</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="LBL_End">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Start">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_End_Title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>End:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Start_Title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Title_Header">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Title:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LBL_Title">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="LBL_Header">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#1A3760</string>
   </property>
   <property name="text">
    <string>CURRENT SCHEDULE DETAILS</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
